src/app/globals.css

```
@@ -1,7 +1,7 @@
 @import 'tailwindcss';
 @import 'tw-animate-css';
-@import "tw-animate-css";
-@import "shadcn/tailwind.css";
+@import 'tw-animate-css';
```

Stylelint (17.6.0)

[error] Unexpected duplicate `@import` rule tw-animate-css (no-duplicate-at-import-rules)

(no-duplicate-at-import-rules)